### ISSUE TITLE
Feedback on Guideline “Visual contrast of text” [Deque feedback]

### ISSUE THREAD

**Issue by melaniephilipp** [w3c/silver#382]
Referencing Github issue w3c/wcag3#307 : Could the [Color on the Web Community Group](https://www.w3.org/community/colorweb/) be of help in facilitating a 3rd party review and validation of the research and resulting model?

Referencing Method “Font characteristics contrast”: The method requires assessing a page by applying a lookup table approach that depends on font face, size, and weight (and maybe the use case), for every variation of text and background on a page. (https://www.myndex.com/APCA/).  The current state of tools would make this a high manual burden.  If tools do not materialize to perform this assessment through automation, this method will be an undue manual burden for assessors.

**Comment by Myndex** [comment]
Hi @melaniephilipp thank you for the comment,

You said:
> ._..for every variation of text and background on a page..._

No, only the lowest scoring case. It is functionally not that different than WCAG 2.x, but ultimately is substantially more flexible. The lookup table indicates the minimum of the range, but there are simplified methods to achieve this. And a key important factor is "use case".

To make it simple, here are "key" levels and what they mean. These key APCA levels are the G series undergoing some final evaluations, and are different from the FPWD:

### APCA Key Levels (Series G)
**75:** The minimum contrast for columns of body text, using a normal weight (400) or heavier font.
**60:** The minimum contrast for other, larger, content text (24px normal or 14 px bold)
45: The minimum contrast for very large bold text, otherwise only non-content text and non text elements should be here or lower.
30: The "recommended" cut off for any kind of text whatsoever, including copyright, placeholder, etc.
20: The absolute hard cut off for any kind of text (automatic fail for any text below 20, might become 25, pending)
10: The absolute hard cut off for any non text element that needs to be discerned to any degree. (might become 15, pending).

So, let's just consider the top two, **75 and 60** — these are sort of like the "two" contrasts of WCAG 2.x, one for big text one for small, and we can stop here. For the highest score, maintain the following, (size/weight relative to a qualified reference font like Helvetica):

- Columns of body text, and smaller text: no less than L<sup>c</sup> 75, no smaller than 16px, no lighter than 400 weight.
- Other content text (menus, headlines, action items): no less than L<sup>c</sup> 60, no smaller than 24px, no lighter than 400 weight.

And done. This is essentially emulating the WCAG 2.x conformance, which was developed over 12 years ago, when fonts were "Verdana" and only available in 400 and 700.

However, this is constrictive to the designer. The **_full_** new contrast method opens up far wider possibilities for design, rooted in actual vision science, not to mention centuries of classical design for print. In traditional print, small thin light grey text was unthinkable. Yet today it is the core of the very bad design choices permeating the web, and making the web literally unreadable.

And you do not have to fully test all text on the page, just the lowest contrast use case(s) to find the score.

WCAG 2.x requires that all text on a page meet the WCAG 2.x standard, which in some cases is more than enough, and in other cases is woefully inadequate. WCAG 2.x is no longer relevant for today's webcontent, as it was designed around two font weights, and web type fonts that were designed for display use such as Verdana.

### Adding Flexibility while Maintaining Readability
If you want to allow designer to use a thin font, 200 or 300 weight, then you can not use such a simplistic approach as WCAG 2.x does. WCAG 3.0 and the APCA does not mandate brute-forced values for cases like non-text, and non-content text (like copyrights or by lines). Instead, key content text must be at the appropriate level for fluent readability. Things like copyright notices can be at the level for "spot reading" which is substantially relaxed.

WCAG 2.x did not define a minimum size, which has resulted in some very bad content designs. WCAG 3.0 does, but does so based on use case, which vastly improves design flexibility.

It is nearly trivial for an automated tool to be  created that automatically tests an entire page and all text therein to meet the calculated minimum values, and there are some in the works right now.

But again, for the manual assessment, it is based around the lowest contrast sample as presently stated.

I hope this answers your question, and please feel free to followup.


Thank you,

Andy


_Andrew Somers_
_W3 AGWG Invited Expert_
_Myndex Color Science Researcher_
_Inventor of SAPC & APCA Contrast_

**Comment by Myndex** [comment]
@melaniephilipp 
To add: please see the SIMPLIFIED too at:  https://www.myndex.com/APCA/simple

**Comment by Myndex** [comment]
Hi @melaniephilipp

I just realized I missed answering your first question:

> _Referencing Github issue #272 : Could the Color on the Web Community Group be of help in facilitating a 3rd party review and validation of the research and resulting model?_

I am also a member of the Color on the Web Community Group, and many (if not most) members there are already very familiar with my work, and I have received substantial positive feedback, some of which is found in issue #695 which I linked elsewhere.

There has been some internal discussion in Silver/AGWG regarding forming an outside panel of some of the recognized experts in the related fields.

Nevertheless, the method is directly based on highly regarded research, and the results are easy to see, a prima facie evidence, especially in comparisons.

WCAG 2.x issue #695 has experimental results posted, there are many experiments and results posted on the Myndex.com site, and here are two Gists that do a quick compare with examples between WCAG 2.x and 3.0

[**Orange You Wondering About Contrast?**](https://gist.github.com/Myndex/1dadb6dcac596f1cd7a5686a076f697f)

[**The Lighter Side of Dark Backgrounds**](https://gist.github.com/Myndex/c30dba273aa5eca426ad9f5200917c9d)


Thank you,

Andy


_Andrew Somers_
_W3 AGWG Invited Expert_
_Myndex Color Science Researcher_
https://www.myndex.com/APCA/simple

### _APCA — THE REVOLUTION WILL BE READABLE_

**Comment by lauracarlson** [comment]
Thank you for your comment. Project members are working on your comment. You may see discussion in the comment thread and we may ask for additional information as we work on it. We will mark the official response when we are finished and close the issue.

**Comment by ChrisLoiselle** [comment]
@jspellman This relates to issue #272 and issue #370 , this is ready for survey.

**Comment by jspellman** [comment]
@ChrisLoiselle, I am removing the "ready for survey" label because there isn't a draft summary response that is addressed to the commenter.  See my separate email.

**Comment by ChrisLoiselle** [comment]
DRAFT RESPONSE:  The questions raised in this issue relate to issue #272 and issue #370 . The main question was answered in this issue thread by Andy on Feb 27.  I hope this answers your question, if it does not, please feel free to follow-up.
Thank you,
Chris

**Comment by Myndex** [comment]
The WCAG 3 repo has an issue thread containing a list of [independent third party reviews of APCA](https://github.com/w3c/wcag3/issues/29), including journal published reviews.

**Regarding automation:** [Draft Bronze at the IRT-RC](https://readtech.org/ARC/tests/bronze-simple-mode/?tn=criterion#conformance-minimums) is a threshold-level only, similar to WCAG 2. This is trivial for existing automation, and many tool makers have already incorporated it as beta for testing and evaluation purposes. 

No level requires manual lookup tables, higher level tools report minimums as part of the tool, and is trivial to automate. **The entire system is designed with automation in mind, and as such is beneficial to companies that rely on automation in their business model.**

**Comment by fstrr** [comment]
This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines \(WCAG\) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).
